src/turnkeyml/common/filesystem.py

```diff
@@ -322,10 +322,10 @@ class Keys:
     # ONNX model input tensor dimensions
     ONNX_INPUT_DIMENSIONS = "onnx_input_dimensions"
     # List of all build stages in the Sequence
-    ALL_BUILD_STAGES = "all_build_stages"
+    ALL_BUILD_STAGES = "selected_stages"
```

Comment:
The design pattern in `filesystem.Keys` is that the variable name and string value match. If you're going to change a value, please change the key as well.

Reply:
Changed as suggested